src/v2/components/PhotoGallery/GalleryContent.tsx

```
@@ -0,0 +1,121 @@
+import { usePullToRefresh } from '../../hooks/usePullToRefresh';
+import { useI18n } from '../../i18n/store';
+import type { Photo } from '../../types/photo';
+import LocationGroupHeader from '../LocationGroupHeader';
+import PhotoGrid from '../PhotoGrid';
+import { useGroupInView } from './useGroupInView';
+
+interface GroupedPhotos {
+  photos: Photo[];
+  location: string;
+  date: string;
+  locationDetail: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)